hudi-flink-datasource/hudi-flink/pom.xml

```diff
@@ -45,8 +45,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
```

Comment:
I think we should keep it personally https://github.com/apache/hudi/blob/master/hudi-flink-datasource/hudi-flink/pom.xml so that if we make a change to java version in root pom